src/NUnitConsole/nunit3-console/Program.cs

```diff
@@ -134,7 +135,9 @@ public static int Main(string[] args)
                     }
                     catch (Exception ex)
                     {
-                        OutWriter.WriteLine(ColorStyle.Error, ex.ToString());
+                        OutWriter.WriteLine(ColorStyle.Error, ExceptionHelper.BuildMessage(ex));
+                        OutWriter.WriteLine();
+                        OutWriter.WriteLine(ColorStyle.Error, ExceptionHelper.BuildStackTrace(ex));
```

Comment:
This is a very top-level try catch, which would presumably handle exceptions which occur within the console, rather than engine. Seemed sensible to additionally add the functionality here.